app/DateTimePicker/time-picker.scss

```diff
@@ -4,38 +4,38 @@
     &.padding {
         padding: 16px;
     }
-    .picker-time-icon {
-        position: absolute;
-        top: 4px;
-        right: 2px;
-        cursor: pointer;
-        padding-right: 0.8rem;
+    span {
+        display: flex;
+        align-items: center;
 
-        svg {
-            @include themify($themes) {
-                fill: themed('ChartHistoryPickerIcon');
+        .picker-time-icon {
+            right: 2px;
+            cursor: pointer;
+            padding-right: 0.8rem;
+
+            svg {
+                @include themify($themes) {
+                    fill: themed('ChartHistoryPickerIcon');
+                }
             }
         }
     }
     #time-picker-input.time-picker-input {
-        height: 32px;
-        padding: 0 8px;
-        border-radius: $border-radius;
-        box-sizing: border-box;
+        font-family: 'IBM Plex Sans';
```

Comment:
Would be great to know why we need to change this? Also, if we change this, we need to set the fallback font as well, for example, sans-serif.